packages/runtime/src/runtime/screen.tsx

```diff
@@ -34,6 +35,7 @@ export const EnsembleScreen: React.FC<EnsembleScreenProps> = ({
   const routeParams = useParams(); // route params
   const params = new URLSearchParams(search); // query params
   const queryParams: { [key: string]: unknown } = Object.fromEntries(params);
+  const [apis, setApis] = useState<EnsembleAPIModel[]>([]);
```

Comment:
Use a map for easy lookup.

Reply:
@evshi sorry, but I did not understand your suggestion about use of map

Reply:
Instead of an array of API models, use a map.